DebugSwift/Sources/Features/App/Push Notifications/Helpers/LocalNotificationSimulatorView.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  File.swift
+//  
+//
+//  Created by Morgan Dock on 12/13/24.
+//
+
+import SwiftUI
+import UserNotifications
+
+@available(iOS 13.0, *)
+struct LocalNotificationSimulatorView: View {
+    @State private var title: String = ""
+    @State private var subtitle: String = ""
+    @State private var textBody: String = ""
+    @State private var badge: String = ""
+    @State private var sound: Bool = true
+    @State private var timeInterval: Double = 5
+
+    var body: some View {
+        NavigationView {
+            Form {
+                Section(header: Text("Notification Content")) {
+                    TextField("Title", text: $title)
+                    TextField("Subtitle", text: $subtitle)
+                    TextField("Body", text: $textBody)
+                    TextField("Badge (number)", text: $badge)
+                        .keyboardType(.numberPad)
+                    Toggle("Play Sound", isOn: $sound)
+                }
+
+                Section(header: Text("Trigger")) {
+                    HStack {
+                        Text("Time Interval")
+                        Spacer()
+                        Text("\(Int(timeInterval)) seconds")
+                    }
+                    Slider(value: $timeInterval, in: 1...60, step: 1)
+                }
+
+                Section {
+                    Button(action: scheduleNotification) {
+                        Text("Schedule Notification")
```

Comment:
localizations

Reply:
@MaatheusGois  why does the DebugSwift project have English as the base localization but also has an en.lproj duplicate of the Base lproj ? we shouldn't need that duplication right?